python/mqt/syrec/simulation_view/dialogs/simulation_run_editor_dialog.py

```
@@ -0,0 +1,1551 @@
+            )
+            if qreg_name == associated_qreg_name and not qubit_values_groupbox.isVisible():
+                is_any_qubit_values_groupbox_collapsed = True
+                qubit_values_groupbox.setVisible(True)
+                qubit_values_toggle_button.setText(TOGGLE_OUTPUT_STATE_QUBIT_VALUES_EDIT)
+                qreg_input_state_input_field.setEnabled(False)
+                qreg_expected_output_state_input_field.setEnabled(False)
+            else:
+                qubit_values_groupbox.setVisible(False)
+                qubit_values_toggle_button.setText(EDIT_OUTPUT_STATE_QUBIT_VALUES)
+                qreg_input_state_input_field.setEnabled(True)
+                qreg_expected_output_state_input_field.setEnabled(qreg_expected_output_state_input_field.text() != "")  # noqa: PLC1901
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC1901`)

Remove unused `noqa` directive

(RUF100)